bson/src/main/scala/dao/BsonDao.scala

```diff
@@ -157,15 +157,28 @@ abstract class BsonDao[Model, ID](db: => DB, collectionName: String)(implicit mo
     }
   }
 
-  private val (maxBulkSize, maxBsonSize): (Int, Int) =
```

Comment:
I made this because the variable metadata inside MongoConnection is now declared : 
`@volatile private[api] var metadata: Option[ProtocolMetadata] = None`
We can't access anymore to maxBulkSize and maxBsonSize directly.